static/third/bootstrap/js/bootstrap.js

```diff
@@ -2107,6 +2110,8 @@
   , mouseenter: function (e) {
       this.$menu.find('.active').removeClass('active')
       $(e.currentTarget).addClass('active')
+      this.$menu.find(".unsubscribed_icon.show").removeClass("show").addClass("hide")
+      this.$menu.find(".active .unsubscribed_icon").removeClass("hide").addClass("show")
```

Comment:
Do we actually need this to patch bootstrap?  Seems like we might be able to do CSS that just does something with `.active .unsubscribed_icon` to show/hide the icon...

Reply:
Ahh, Great Thanks :smile: I wasn't thinking non-JavaScript approach, but with CSS it becomes too simple.